src/Provider/MapQuest/composer.json

```diff
@@ -18,7 +18,7 @@
     },
     "require-dev": {
         "phpunit/phpunit": "6.3.*",
-        "geocoder-php/provider-integration-tests": "^1.0",
+        "geocoder-php/provider-integration-tests": "^1.1",
```

Comment:
Yes, I was going to do this change when you finially tagged `geocoder-php/provider-integration-tests`. Good thing that you now have added a new tag there.